ginivision/src/androidTest/java/net/gini/android/vision/util/UriHelperTest.java

```diff
@@ -0,0 +1,127 @@
+package net.gini.android.vision.util;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import android.content.Context;
+import android.net.Uri;
+import android.support.annotation.NonNull;
+import android.support.test.InstrumentationRegistry;
+import android.support.test.runner.AndroidJUnit4;
+import android.support.v4.content.FileProvider;
+
+import net.gini.android.vision.test.Helpers;
+
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import java.io.File;
+import java.io.IOException;
+
+/**
+ * Created by Alpar Szotyori on 28.11.2017.
+ *
+ * Copyright (c) 2017 Gini GmbH.
+ */
+
+@RunWith(AndroidJUnit4.class)
+public class UriHelperTest {
+
+    private static final String TEST_FILE = "invoice.jpg";
+
+    @BeforeClass
+    public static void setUpClass() throws Exception {
+        setUpFileProvider();
+    }
+
+    private static void setUpFileProvider() throws IOException {
+        final File fileProviderDir = getFileProviderDir();
```

Comment:
Since you are using the file provider dir in your test cases and it is always the same, would it make sense to create an instance property for that and initialize it here?